fc/parsing/actions.py

```diff
@@ -1451,7 +1616,7 @@ def _list_parameters(self):
 
             # Parameters are inputs that aren't states, and have a constant RHS
             eq = self.model.get_definition(variable)
-            if isinstance(eq.lhs, VariableDummy) and len(eq.rhs.atoms(VariableDummy)) == 0:
+            if not eq.lhs.is_Derivative and len(eq.rhs.atoms(VariableDummy)) == 0:
```

Comment:
Just a general sense that checking a property is better than using type checking?
Been a while since I saw this code so learned a bit more about cellmlmanip / sympy